src/PureScript/Backend/Chez/Syntax.purs

```diff
@@ -76,13 +76,13 @@ data ChezImportSet
 
 type LibraryBody =
   { definitions :: Prim.Array ChezDefinition
-  , exprs :: Prim.Array ChezExpr
+  , expressions :: Prim.Array ChezExpr
   }
 
 data ChezDefinition
   = DefineValue Prim.String ChezExpr
   | DefineCurriedFunction Prim.String (NonEmptyArray Prim.String) ChezExpr
-  | DefineUncurriedFunction Prim.String (NonEmptyArray Prim.String) ChezExpr
+  | DefineUncurriedFunction Prim.String (Prim.Array Prim.String) ChezExpr
```

Comment:
Uncurried functions can have zero arguments, as implied by the `Fn0` type: https://pursuit.purescript.org/packages/purescript-functions/6.0.0/docs/Data.Function.Uncurried#t:Fn0

In Chez, they'd look like this:
```cs
> (define fn0 (lambda () 0))
> (fn0)
0
```